### ISSUE TITLE
`check_autoload_position()` for `ModLoaderStore` not able to log error

### ISSUE THREAD

**Issue by KANAjetzt** [GodotModding/godot-mod-loader#262]
Due to the significant dependency on `ModLoaderStore`, the current implementation of `_ModLoaderGodot.check_autoload_position()` is unable to log the intended error message.

```
`_ModLoaderGodot.check_autoload_position()`
  `ModLoaderLog.fatal()`
     `ModLoaderLog._log()` 
        -> `ModLoaderLog._is_mod_name_ignored()`
 ```
 
`ModLoaderLog._is_mod_name_ignored()` retrieves an array of ignored mod names from `ModLoaderStore`:
 
 ```gdscript
 static func _is_mod_name_ignored(mod_name: String) -> bool:
	var ignored_mod_names := ModLoaderStore.ml_options.ignored_mod_names_in_log as Array

	if not ignored_mod_names.size() == 0:
		if mod_name in ignored_mod_names:
			return true
	return false
 ```

**Comment by KANAjetzt** [comment]
Because this is an error that should not be ignored and it also doesn't need to be stored if the mod loader is not working anyway, this solution seems acceptable:

```gdscript
push_error(final_message)
ModLoaderLog._write_to_log_file(final_message)
ModLoaderLog._write_to_log_file(JSON.print(get_stack(), "  "))
assert(false, final_message)
```

 Calling the individual components of the fatal log separately.

**Comment by KANAjetzt** [comment]
closed by #264
